charts/__tests__/gardener-dashboard/clusterrolebinding.spec.js

```diff
@@ -9,7 +9,7 @@
 const { basename } = require('path')
 const { helm } = fixtures
 
-const chart = basename(__dirname)
+const chart = basename(__dirname) + '/charts/application' // TODO discuss
```

Comment:
I would propose to change the function `renderChartTemplates` in `__fixtures__/helm.js` like this:
```js
function renderTemplatesFn (...paths) {
  const chart = paths[0]
  const cwd = path.resolve(__dirname, '..', ...paths)
  let name, defaults
  try {
    { defaults = {}, releaseName } = require('./' + chart)
    { name = releaseName } = yaml.load(fs.readFileSync(path.join(cwd, 'Chart.yaml'), 'utf8'))    
  } catch (err) {
    assert.fail(`Invalid chart "${chart}"`)
  }
  return async (templates, values) => {
    defaultsDeep(values, defaults)
    ...
  }
}
```
and use it like this:
```js
const renderTemplates = helm.renderTemplateFn(basename(__dirname), 'charts', 'application')
```